Idno/start.php

```diff
@@ -94,9 +94,15 @@ function &loader()
 
 // Bonita is being used for templating
     $known_loader->registerNamespace('Bonita', dirname(dirname(__FILE__)) . '/external/bonita/includes');
+//PSR Log Interface    
+    include dirname(dirname(__FILE__)) . '/external/Psr/Log/LoggerInterface.php';
+    include dirname(dirname(__FILE__)) . '/external/Psr/Log/AbstractLogger.php';
+    include dirname(dirname(__FILE__)) . '/external/Psr/Log/LogLevel.php';
+
```

Comment:
This should not be necessary - you should be able to add external/PSR with the loader's registerNamespace function instead.